runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -204,4 +205,10 @@
   ExecutionMode getExecutionModeForBatch();
 
   void setExecutionModeForBatch(ExecutionMode executionMode);
+
+  @Description("Directory containing Flink YAML configuration files.")
+  @Nullable
+  String getFlinkConfigDir();
+
+  void setFlinkConfigDir(@Nullable String value);
```

Comment:
As discussed in the JIRA, the better place for this is the JobServer configuration. Sorry for the hassle.

Reply:
Sounds reasonable. I will move it their.